lib/vmdb/loggers.rb

```diff
@@ -82,15 +82,22 @@ def self.create_loggers
 
       configure_external_loggers
     end
-    private_class_method :create_loggers
 
-    def self.create_multicast_logger(log_file_path, logger_class = VMDBLogger)
+    private_class_method def self.create_multicast_logger(log_file_path, logger_class = VMDBLogger)
       logger_class.new(log_file_path).tap do |logger|
-        logger.extend(ActiveSupport::Logger.broadcast($container_log)) if ENV["CONTAINER"]
+        logger.extend(ActiveSupport::Logger.broadcast($container_log)) if $container_log
         logger.extend(ActiveSupport::Logger.broadcast($journald_log))  if $journald_log
       end
     end
-    private_class_method :create_multicast_logger
+
+    private_class_method def self.create_container_logger
+      return unless ENV["CONTAINER"]
+
+      require "manageiq/loggers/container"
+      ManageIQ::Loggers::Container.new
+    rescue LoadError
```

Comment:
Why silently fail on this one?  It seems like it would be a problem for containers.

Reply:
OH yeah, I just did a copy paste from the journald one...I can remove that

Reply:
:+1: we're catching LoadError in case developers don't have the journald gem installed locally, not needed for the container logger